newrelic/hooks/framework_graphql.py

```diff
@@ -105,7 +105,6 @@ def bind_operation_v2(exe_context, operation, root_value):
 def wrap_execute_operation(wrapped, instance, args, kwargs):
     transaction = current_transaction()
     trace = current_trace()
```

Comment:
Do we need a no harm test for this case where we are not in a transaction?

Reply:
Added one.